view/templates/head.tpl

```diff
@@ -44,6 +44,7 @@
 <script type="text/javascript" src="view/asset/imagesloaded/imagesloaded.pkgd.min.js"></script>
 <script type="text/javascript" src="view/js/acl.js" ></script>
 <script type="text/javascript" src="view/asset/base64/base64.min.js" ></script>
+<script type="text/javascript" src="view/asset/dompurify/dist/purify.min.js"></script>
```

Comment:
Can we use `App->AddFooterScript` instead?

Reply:
Why? This should be used everytime a tooltip or hovercard appears, which is very often in Friendica. So I added it to the "default" place for javascript libraries. The `registerFooterScript` is only used by addons yet.

Reply:
Any script/library that only acts after page loading should be placed in the footer to speed up the first render.

I started using it in addons because they change more often than the base page, but ideally all the scripts (including `main.js`) probably should be referenced in the footer.

Reply:
That's true of course